examples/test_delete_me/main.tf

```diff
@@ -0,0 +1,87 @@
+terraform {
```

Comment:
Is this intended to become a real example or shall we remove it?

Reply:
my fault, i did not intend to commit this file. Ill remove it.